buildSrc/src/main/kotlin/com.qpay.java.base.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id("com.qpay.java.tests")
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_21
+    targetCompatibility = JavaVersion.VERSION_21
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(21)
+    }
+}
+
+configurations {
+    compileOnly {
+        extendsFrom(configurations.annotationProcessor.get())
+    }
+}
+
+dependencies {
+    annotationProcessor("org.projectlombok:lombok")
+
+    implementation("org.springframework.boot:spring-boot-starter-web")
```

Comment:
This plugin is for java. For spring boot, you can create separate one but no need now.
Also it's not a good idea to add spring-boot-web to base as it makes us use exactly this framework. But probably some of the services will use spring webflux or micronaut.